spec/order_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "order"
+
+describe Order do
+    describe "#take_order" do
+      it "exists" do
+        expect(subject).to respond_to(:take_order)
```

Comment:
When testing, if we test what take_order does, we implicitly test that take_order is a valid method. So there's no need to write this kind of test.